packages/minter-contracts/src/fa2-interface.ts

```diff
@@ -90,3 +90,25 @@ export async function removeOperator(
   await op.confirmation(3);
   $log.info(`consumed gas: ${op.consumedGas}`);
 }
+
```

Comment:
Consider using view entry points (if you have them defined) and taquito view functionality instead of directly accessing the storage

Reply:
I have not defined the `token_metadata` view entrypoint. I will make a ticket to do this for future work and avoid accessing storage directly in all tests.

Reply:
I believe that the most recent version of FA2 makes this entry point optional.
Maybe you should keep accessing `token_metadata` directly (as permitted by the FA2 standard) if you do not plan to add view entry point in future. In other word, do not add contract code just for testing purposes only